src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/PostgreSqlEngine.java

```diff
@@ -532,11 +546,12 @@ protected void dropColumn(DbEntity entity, String... columns) throws DatabaseEng
         }
         removeColumns.add(join(cols, ","));
 
+        final AtomicReference<Statement> queryStatement = new AtomicReference<>();
```

Comment:
Why do we need the atomic references here and in the other lines?

Reply:
Ok, removed! Now the new method `com.feedzai.commons.sql.abstraction.engine.impl.PostgreSqlEngine#executeUpdateQuery` closes the `Statement`.